src/sentry/api/endpoints/organization_teams.py

```diff
@@ -49,6 +49,7 @@ class TeamPostSerializer(serializers.Serializer):
         },
     )
     idp_provisioned = serializers.BooleanField(required=False, default=False)
+    set_admin = serializers.BooleanField(required=False, default=False)
```

Comment:
would it be more coherent to have a `team_role` query param that defaults to `contributor` but can also be set to `admin`?

Reply:
I like having the qparam as a bool over a string b/c its a binary choice and you dont have to validate `team_role`. I changed the name to `setTeamAdmin` but it's open to anything that makes more sense

Reply:
okay. i was also wondering in the case where we somehow expand team roles but this works